src/Handlers/BookingsStateStatusTransitionsHandler.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace RebelCode\Bookings\WordPress\Module\Handlers;
+
+use Dhii\Exception\CreateInvalidArgumentExceptionCapableTrait;
+use Dhii\I18n\StringTranslatingTrait;
+use Dhii\Invocation\InvocableInterface;
+use Dhii\Util\Normalization\NormalizeArrayCapableTrait;
+use Psr\EventManager\EventInterface;
+use stdClass;
+use Traversable;
+
+/**
+ * Handler for adding transitions information to state on bookings page.
+ * It will add map of available statuses transitions and transitions translations.
+ *
+ * @since [*next-version*]
+ */
+class BookingsStateStatusTransitionsHandler implements InvocableInterface
+{
+    /* @since [*next-version*] */
+    use NormalizeArrayCapableTrait;
+
+    /* @since [*next-version*] */
+    use CreateInvalidArgumentExceptionCapableTrait;
+
+    /* @since [*next-version*] */
+    use StringTranslatingTrait;
+
+    /**
+     * Map of transitions that should be applied when changing status.
+     *
+     * @since [*next-version*]
+     *
+     * @var array|Traversable|stdClass
+     */
+    protected $statusTransitions;
+
+    /**
+     * List of labels for transitions.
+     *
+     * @since [*next-version*]
+     *
+     * @var array|Traversable|stdClass
+     */
+    protected $transitionsLabels;
+
+    /**
+     * StatusesHandler constructor.
+     *
+     * @since [*next-version*]
+     *
+     * @param array|Traversable|stdClass $statusTransitions Map of transitions for changing statuses.
+     * @param array|Traversable|stdClass $transitionsLabels Map of transitions for changing statuses.
```

Comment:
It appears that at least this parameter needs to be a map, therefore `Traversable` is no longer sufficient. Use [`MapInterface`][] for cases like this. Also please revise other parameters, including in other handlers.

[`MapInterface`]: https://github.com/Dhii/collections-interface/blob/develop/src/MapInterface.php#L13